spec/models/comment_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  @user1 = User.new(name: 'Nabeel', photo: 'image url here', bio: 'Front end developer')
+  @user2 = User.new(name: 'Ahmed', photo: 'image  url here', bio: 'Back end developer')
+  @post1 = Post.new(author: @user1, title: 'Hello', text: 'first test post')
```

Comment:
- [ ] @Metaverse-Nabeel Great work on your tests, however, am getting the error below when I run your tests, kindly ensure all tests are running, the issue is caused by using an unknown attribute, instead of `author_id` you have used author.
![image](https://github.com/Metaverse-Nabeel/blog_app/assets/61408860/796f9a24-5179-45e1-8acd-5dfbd9b54458)

Reply:
Hello @danmainah 

I have made the changes to the tests. Now, all tests are working fine.

Regards